apps/Cargo.toml

```diff
@@ -147,9 +148,8 @@ tracing-log = "0.1.2"
 tracing-subscriber = {version = "0.3.7", features = ["env-filter", "json"]}
 websocket = "0.26.2"
 winapi = "0.3.9"
-#libmasp = { git = "https://github.com/anoma/masp", branch = "murisi/masp-incentive" }
-masp_primitives = { git = "https://github.com/anoma/masp", rev = "bee40fc465f6afbd10558d12fe96eb1742eee45c", features = ["transparent-inputs"] }
-masp_proofs = { git = "https://github.com/anoma/masp", rev = "bee40fc465f6afbd10558d12fe96eb1742eee45c", features = ["bundled-prover", "download-params"] }
+masp_primitives = { git = "https://github.com/anoma/masp", branch = "murisi/namada-integration", features = ["transparent-inputs"] }
```

Comment:
let's keep using `rev` - it's easier to manage when the branch changes  (we can note the branch name next to it)

Reply:
done in 389041b73967dbe03a90be2c5d4079a5a516b5ea